README.md

```diff
@@ -4,25 +4,122 @@ Anaconda Recipes
 This is a Continuum managed repository for recipes used in the
 free [Anaconda Python distribution](https://www.continuum.io/downloads).
 
-The machines (and additional software installed) to build these conda
-recipes are as follows:
+Building Conda Recipes
+================
 
-Linux:
-  * we use sepatate 32 and 64-bit machines
-  * CentOS release 5.11 (Final)
-  * gcc 4.4.7 (`yum install gcc44`, link `/usr/bin/gcc` to `gcc44`)
-  * GNU Fortran (GCC) 4.4.7
 
-Mac OS X (64-bit):
-  * Version 10.7.5
-  * gcc 4.2.1
-  * GNU Fortran (GCC) 4.2.3
+##### Linux (use separate 32 and 64-bit machines):
+
+Machine and additional software:
+ * CentOS release 5.11 
+ * gcc 4.4.7 (`yum install gcc44`, link `/usr/bin/gcc` to `gcc44`)
+ * GNU Fortran (GCC) 4.4.7
+
+We use a Docker image, “conda-builder”, to build the packages.
+
+Install Docker: 
+ 
+ Linux: 
+  * https://docs.docker.com/linux/step_one/ 
+  * Start the docker service: sudo service docker start
+ 
+Windows/Mac: 
+  * https://www.docker.com/products/docker-toolbox
+  * Run Docker Quickstart Terminal
+
+Download the following files: 
+
+ * https://github.com/ContinuumIO/docker-images/blob/master/conda_builder_linux/docker_wrapper.sh
+ * (64-bit) https://github.com/ContinuumIO/docker-images/blob/master/conda_builder_linux/start_cpp98.sh
+ * (32-bit) https://github.com/ContinuumIO/docker-images/blob/master/conda_builder_linux/start_cpp98_32.sh
```

Comment:
It might be worth mentioning that cpp98 indicates that the script sets the default C++ ABI to the older format.  This does not preclude compiling C++11 or C++14 code, it only implies the older ABI.